modules/account-lib/src/index.ts

```diff
@@ -80,6 +80,9 @@ export { Bsc };
 import * as Polygon from '@bitgo/sdk-coin-polygon';
 export { Polygon };
 
+import * as Sui from '@bitgo/sdk-coin-sui';
```

Comment:
Is this required for bgms or can we keep it only in `bitgo`?

Reply:
this should be required for bgms